src/SDKs/CognitiveServices/dataPlane/Vision/FormRecognizer/FormRecognizer/Microsoft.Azure.CognitiveServices.FormRecognizer.csproj

```diff
@@ -0,0 +1,20 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <!-- Please do not move/edit code below this line -->
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
+  <!-- Please do not move/edit code above this line -->
+
+  <ItemGroup>
+    <Folder Include="Properties\" />
+  </ItemGroup>
+
+  <PropertyGroup>
+    <TargetFrameworks>$(SdkTargetFx)</TargetFrameworks>
```

Comment:
Please refer [this](https://github.com/Azure/azure-sdk-for-net/blob/6e48acd9b5ad03fdf591f058417b434fdbdfd023/src/SDKs/CognitiveServices/dataPlane/Vision/ContentModerator/ContentModerator/Microsoft.Azure.CognitiveServices.ContentModerator.csproj) as an example and add the remaining attributes (Assembly version, package release notes, etc
Also, please set the assembly version to `0.8.0-preview`

Reply:
done.

Reply:
done.

Reply:
done.